pkg/morserino_com/listen_com.go

```diff
@@ -24,67 +24,126 @@ THE SOFTWARE.
 
 import (
 	"fmt"
+	"io"
 	"log"
 	"strings"
+	"testing/iotest"
 
-	"morserino_display/pkg/morserino_console"
-
+	"github.com/on4kjm/morserino_display/pkg/morserino_console"
 	"go.bug.st/serial"
 )
 
+// Channel used to forward data received on the serial port to the display modules
+var MessageBuffer = make(chan string, 10)
+
 // Main listen function with display to the console
 func Listen_console(morserinoPortName string, genericEnumPorts comPortEnumerator) error {
 
-	//Port parameters for a Morserino
-	mode := &serial.Mode{
-		BaudRate: 115200,
-		Parity:   serial.NoParity,
-		DataBits: 8,
-		StopBits: serial.OneStopBit,
+	//If requested, use the simulator instead of a real Morserino
+	if strings.HasPrefix("SIMULATOR", strings.ToUpper(morserinoPortName)) {
+		TestMessage := "cq cq de on4kjm on4kjm = tks fer call om = ur rst 599 = hw? \n73 de on4kjm = <sk> e e"
+		Listen(iotest.OneByteReader(strings.NewReader(TestMessage)))
+		return nil
```

Comment:
```suggestion
		return Listen(iotest.OneByteReader(strings.NewReader(TestMessage)))
```

Reply:
Indeed: the error possibly returned by Listen() should be pushed to the caller.